jq_dict_parser.go

```diff
@@ -27,20 +26,16 @@ func NewJQDictParser(dict map[string]string) map[string]JQFieldParser {
 	return output
 }
 
-func (p JQDictParser) Run(data string) (map[string]string, error) {
+func (p JQDictParser) Run(data string) map[string]string {
 	output := make(map[string]string)
 	for key, expression := range p {
-		jqRes, err := expression.Run(data)
-		if err != nil {
-			log.Warn().Str("key", key).Err(err).Msg("error running jq expression")
-			continue
-		}
+		jqRes := expression.Run(data)
 		if jqRes == "null" {
```

Comment:
Checking for `null` is actually not the right thing to do here like the lines below say. There is a much better way to do this that is going to be in a follow-up PR.